packages/sdk-core/src/repository/HypercertOperationsImpl.ts

```
@@ -856,38 +879,76 @@ export class HypercertOperationsImpl extends EventEmitter<HypercertEvents> imple
     try {
-      const existing = await this.get(hypercertUri);
-      const existingEvidence = (existing.record.evidence as HypercertEvidence[]) || [];
-      const updatedEvidence = [...existingEvidence, ...evidence];
+      const { subjectUri, content, ...rest } = evidence;
+      const subject = await this.get(subjectUri);
+      const createdAt = new Date().toISOString();
 
-      const result = await this.update({
-        uri: hypercertUri,
-        updates: { evidence: updatedEvidence },
+      let evidenceContent = await this.getHypercertEvidenceContent(content);
```

ESLint

[error] 'evidenceContent' is never reassigned. Use 'const' instead.

(prefer-const)